src/infohud.asm

```diff
@@ -561,35 +561,43 @@ status_roomstrat:
     RTS
 
   .status_room_table
-    dw status_mbhp
+    dw status_tacotank
+    dw status_gateglitch
```

Comment:
Gate Glitch added, Taco Tank is just a placeholder.
Once Taco Tank is done, I plan to merge in this pull request.

I'm also planning to make a new .asm file just for the infohud modes and room strats, since this file is a bit large.